db/connect.js

```diff
@@ -0,0 +1,8 @@
+const mongoose = require('mongoose')
+const url = 'mongodb://127.0.0.1:27017/volunteer'
+
+const connectDB = (url) => {
```

Comment:
you are currently just using url above so lets just change this for now
```suggestion
const connectDB = () => {
```